From 5fe78fd41bf59108111b4469b0e022ac4cde50da Mon Sep 17 00:00:00 2001
From: Michel Alexandre Salim <salimma@fedoraproject.org>
Date: Fri, 2 Nov 2012 11:21:59 +0700
Subject: [PATCH] Remove lid close settings

These have been removed from gnome-settings-daemon master (pre-3.8)
as the functionality is now handled by systemd
---
 gtweak/tweaks/tweak_shell.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/gtweak/tweaks/tweak_shell.py b/gtweak/tweaks/tweak_shell.py
index d80b1e3..d34983a 100644
--- a/gtweak/tweaks/tweak_shell.py
+++ b/gtweak/tweaks/tweak_shell.py
@@ -247,8 +247,6 @@ TWEAK_GROUPS = (
             GSettingsSwitchTweak("org.gnome.shell.calendar", "show-weekdate", schema_filename="org.gnome.shell.gschema.xml"),
             ShowWindowButtons(schema_filename="org.gnome.shell.gschema.xml", size_group=sg),
             GSettingsSwitchTweak("org.gnome.settings-daemon.plugins.power", "lid-close-suspend-with-external-monitor"),
-            GSettingsComboEnumTweak("org.gnome.settings-daemon.plugins.power", "lid-close-battery-action", size_group=sg),
-            GSettingsComboEnumTweak("org.gnome.settings-daemon.plugins.power", "lid-close-ac-action", size_group=sg),
             GSettingsComboEnumTweak("org.gnome.settings-daemon.plugins.power", "button-power", size_group=sg),
             GSettingsSwitchTweak("org.gnome.shell.overrides", "workspaces-only-on-primary"),
             StaticWorkspaceTweak(size_group=sg)),
-- 
1.7.12.1

